src/components/mixins/entity_list.js

```diff
@@ -192,6 +193,27 @@ export const entityListMixin = {
       this.showHeaderMenu()
     },
 
+    onSelectColumn () {
+      const mapOfElements = this.assetMap ? this.assetMap : this.shotMap
+      const listOfElements = Object.values(mapOfElements)
+      const selection = []
+      for (let i = 0; i < Object.keys(mapOfElements).length; i++) {
```

Comment:
In JS, it's better to use `forEach` or `map` functions. For loops are not commonly used.